### PR TITLE
Simple zookeeper role

### PR DESCRIPTION
**Pull request by jamielennox**
If we are happy to use the ubuntu provided java and zookeeper then a
basic install isn't too hard. This is a super basic role that will need
customization to our requirements later. For now the templates are just
copy and pasted the default configs from the ubuntu packages.

Basically I wanted something I could play with for now and tune later.

### CODE REVIEW

File: roles/zookeeper/templates/myid [hunk 1]
```diff
@@ -0,0 +1 @@
+{{ zookeeper_myid | mandatory }}
```
Comment (j2sol) [inline roles/zookeeper/templates/myid]: Where is this going to be defined?

Comment (jamielennox) [inline roles/zookeeper/templates/myid]: I'm thinking on host_var, but i'm not entirely sure.  I was vaguely following [1] but i think that's too complicated until we figure out how we're going to deploy it. 

[1] https://github.com/AnsibleShipyard/ansible-zookeeper/blob/master/templates/myid.j2

Comment (j2sol) [inline roles/zookeeper/templates/myid]: Does this just need to be something unique, or sequential?

Comment (jamielennox) [inline roles/zookeeper/templates/myid]: It needs to be unique and between 1-255 but i don't think it has to be sequential [1]. The reason i'm not sure about the iterate hosts is i'm not sure what happens if a server was to change ids. So for now i figured it was just easiest to flag it as required and let a person deal with the number assignment.

[1] https://zookeeper.apache.org/doc/r3.3.2/zookeeperAdmin.html#sc_zkMulitServerSetup

**Pull request merged**